decentraland/unity-renderer
# chore: add You: text in public channel

## What does this PR change?

Adds a You: text before the message when using Public Channels. 

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=chore/Add-YOU-text-in-public-channel
2. Check that the following conditions are met when sending messages:

- Public chat:
Public Message Sent=>YOU:
Public Message Received=>Username#123:
Private Message Sent=>To Username#123:
Private Message Received=> From Username#123:

- Private chat:
MessageSent=>Sin nombre previo al mensaje
MessageReceived=>Sin nombre previo al mensaje

- Whisper:
Message Sent=>To Username#123:
Message Received=>From Username#123:

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
